test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/WorkbenchKieServerScenarioIntegrationTest.java

```diff
@@ -41,6 +43,7 @@
 import org.kie.server.controller.client.KieServerControllerClient;
 
 @Category(Smoke.class)
+@RunWith(SmokeTestRunner.class)
```

Comment:
Adding the category and the smoke test runner seems to me like a bit verbose. 
Maybe we could use Maven to configure the tests annotated with smoke category by using the smoke test runner.
Or perhaps we could use this custom test runner in all the tests (if and only if we only retry when a deployment timeout exception was raised). What do you think?